ui/config/webpackDevServer.config.js

```diff
@@ -80,66 +103,60 @@ module.exports = function (proxy, allowedHost) {
     hot: true,
     // Use 'ws' instead of 'sockjs-node' on server since we're using native
     // websockets in `webpackHotDevClient`.
-    transportMode: "ws",
+    webSocketServer: "ws", //replaced transportMode v3
```

Comment:
Nit: superfluous comment here and in other places. No need to add comments for changes that describe what's visible via git-diff